admin/javascript/src/LeftAndMain.js

```diff
@@ -227,18 +227,15 @@ $.entwine('ss', function($) {
 			var self = this,
 				basePath = getUrlPath($('base')[0].href);
 
-			// Avoid adding a double slash if the base path is '/'
+			// Remove trailing / from base
 			if (basePath[basePath.length - 1] === '/') {
-				basePath += 'admin';
-			} else {
-				basePath = '/admin';
+				basePath = basePath.substring(0, basePath.length - 1);
```

Comment:
That's encoding assumptions about how SS PHP formats the `<base>` tag. That's overwriteable behaviour, so might not always contain a trailing slash. Can we please use a more targeted `.replace(/\/$, ''/)` there?